### ISSUE TITLE
Add an option to show a clock during gameplay

### ISSUE THREAD

**Issue by zylefk** [TheXTech/TheXTech#760]
Something like this might be useful when players are in fullscreen or for android players, the clock could be visible in one of the corners of the screen

**Comment by Wohlstand** [comment]
Do you mean showing the current time of the device?

**Comment by zylefk** [comment]
> Do you mean showing the current time of the device? That shouldn't be complicated, but this will also have some extra setup like switching between 12/24H format, etc. (By myself I prefer 24h format, some other people do use 12h format with AM/PM)

Yes, as you say, it would be nice to show the time of the device, either in 12h or 24h format.
